### PR TITLE
XIVY-13510 Update to latest deps

### PR DESCRIPTION
**Pull request by ivy-lli**
- Use common colors for inscription

### CODE REVIEW

File: playwright/yarn.lock
Comment (ivy-lmu) [inline playwright/yarn.lock:1]: accidentally checked in?

**Review by ivy-lmu — APPROVED** [review]
👍 without the new yarn.lock ;)

File: playwright/yarn.lock
Comment (ivy-lli) [reply playwright/yarn.lock:1]: hm... where did this one come from?!? 🙈

Comment (ivy-lmu) [reply playwright/yarn.lock:1]: I guess running yarn in playwright directory

**Pull request merged**